.github/workflows/live_tests.yaml

```diff
@@ -1,36 +1,52 @@
 name: Tests against a Live Instance
 
 on:
-  # allow manual runs
+  # Allow manual runs
   workflow_dispatch:
 
-  # run every day
+  # Run every day
   schedule:
-    - cron: "0 0 * * *" 
+    - cron: "0 0 * * *"
 
 jobs:
-  test-against-live-instance:
+  test-live-instance:
     runs-on: ubuntu-latest
     env:
+      # Secrets for API and organization
+      REFRESH_TOKEN: ${{ secrets.DEEP_ORIGIN_REFRESH_TOKEN }}
+      ACCESS_TOKEN: ${{ secrets.DEEP_ORIGIN_ACCESS_TOKEN }}
       DEEP_ORIGIN_ORGANIZATION_ID: ${{ secrets.DEEP_ORIGIN_ORGANIZATION_ID }}
       DEEP_ORIGIN_API_ENDPOINT: ${{ secrets.DEEP_ORIGIN_API_ENDPOINT }}
       DEEP_ORIGIN_NUCLEUS_API_ROUTE: ${{ secrets.DEEP_ORIGIN_NUCLEUS_API_ROUTE }}
-      DEEP_ORIGIN_ACCESS_TOKEN: ${{ secrets.DEEP_ORIGIN_ACCESS_TOKEN }}
```

Comment:
we can't create a env variable called this because then the client will use this. we don't want it to -- we want it to read from file